app/views/instructor_students/index.html.erb

```diff
@@ -9,6 +9,8 @@
   <p>Subject:<%= student.subject%></p>
   <p>Created At: <%= student.created_at %></p>
   <p>Updated At:<%= student.updated_at %></p>
+  <p><%= button_to "View #{student.name}", "/students/#{student.id}", method: :get %>
```

Comment:
Is the View button something we were supposed to add? I may have missed that detail.

Reply:
We discussed on zoom - I don't think it's a requirement but added for usability.